api/src/wfl/service/datarepo.clj

```diff
@@ -236,6 +236,11 @@
                    :query-params {:include include}})
         util/response-body-json)))
 
+(defn snapshot-url
+  "Return a link to `snapshot` in TDR UI."
+  [{:keys [id] :as _snapshot}]
+  (datarepo-url "snapshots/details" id))
```

Comment:
Probably spozed to be `(datarepo-url "snapshots" "details" id)`, but OK.

Reply:
I wondered about that, but I did see other usage patterns like this:

```
(def ^:private repository
  "API URL for Data Repo API."
  (partial datarepo-url "api/repository/v1"))
```